lib/resultuploaders/result_uploader.rb

```diff
@@ -73,9 +88,11 @@ def update_file_content(content, r_name)
     end
 
     def url
-      return nil unless @connected_uploaders.key?(:dropbox)
+      @connected_uploaders.values.filter_map(&:url).first
```

Comment:
What about just using `find`?

Reply:
Variants `.map(&:url).find { !_1.nil? }` and `.find { !_1.url.nil? }&.url` looks more complicated to me.

Reply:
Ah, makes sense.